app/_plugins/tags/support_policy_tag.rb

```diff
@@ -18,6 +18,8 @@ def render(context)
 | Version  | Latest Patch  | Released Date | End of Full Support |
 |:--------:|:-------------:|:-------------:|:-------------------:|\n"
       versions.each do |entry|
+        next if entry['label'] == 'unreleased'
```

Comment:
mesh uses `dev` label for the unreleased version: https://github.com/Kong/docs.konghq.com/blob/c2ffad667f0304f3e8015dccf7d09aa5f783e664/app/_data/kong_versions.yml#L413

Do we want to include that in the skip condition or do we want to leave that as is to not change it for them?

Currently this https://deploy-preview-7325--kongdocs.netlify.app/mesh/latest/support-policy/#version-support-for-kong-mesh still renders the unreleased mesh version. 

cc: @lahabana as you've contributed `app/_plugins/tags/support_policy_tag.rb` and might want to have that changed.

Reply:
This can be answered later on and tackled in a separate PR.

Leaving the thread in here for discussion but it shouldn't preclude merging this PR.